feat: suzyq documentation (#253)
FyraLabs/devdocs: pages/suzyqboard/accessing-ccd.mdx

---
title: Accessing Closed Case Debugging
description: Access Closed Case Debugging (CCD) using the SuzyQ Board
---

> When Accessing Closed Case Debugging, the Fyra SuzyQ Board is no different then using a SuzyQable, so you can just follow [MrChromebox's WP disabling page](https://docs.mrchromebox.tech/docs/firmware/wp/disabling.html).

Reviewer: ```suggestion
When Accessing Closed Case Debugging, the SuzyQ Board is no different then using any other SuzyQable. Just follow [MrChromebox's WP disabling page](https://docs.mrchromebox.tech/docs/firmware/wp/disabling.html).
```

> If you wish to dive very deep, [Google's CCD documentation](https://chromium.googlesource.com/chromiumos/platform/ec/+/cr50_stab/docs/case_closed_debugging_gsc.md) is a great resource for that.

Reviewer: ```suggestion
If you want to dive deeper, [Google's CCD documentation](https://chromium.googlesource.com/chromiumos/platform/ec/+/cr50_stab/docs/case_closed_debugging_gsc.md) is a great resource.
```
